src/Goerli-DssSpell.sol

```diff
@@ -16,29 +16,79 @@
 
 pragma solidity 0.6.12;
 // Enable ABIEncoderV2 when onboarding collateral through `DssExecLib.addNewCollateral()`
-// pragma experimental ABIEncoderV2;
+pragma experimental ABIEncoderV2;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
 import { DssSpellCollateralAction } from "./Goerli-DssSpellCollateral.sol";
 
+import { VatAbstract } from "dss-interfaces/dss/VatAbstract.sol";
+import { JugAbstract } from "dss-interfaces/dss/JugAbstract.sol";
+import { IlkRegistryAbstract } from "dss-interfaces/dss/IlkRegistryAbstract.sol";
+import { DaiAbstract } from "dss-interfaces/dss/DaiAbstract.sol";
```

Comment:
```suggestion
import { VatAbstract, JugAbstract, IlkRegistryAbstract, DaiAbstract } from "dss-interfaces/Interfaces.sol";
```
Consider this shorter version.

Reply:
Tal, I think with your way, the flatten will include every interface and will make it messier.

Reply:
Ah ok.